lib/collective.lib.js

```diff
@@ -232,6 +232,15 @@ export const isSelfHostedAccount = c => c.isHost === true && c.type === 'COLLECT
 export const isIndividualAccount = account => ['USER', 'INDIVIDUAL'].includes(account.type);
 
 /* Checks whether an account supports grants */
-export const accountSupportsGrants = (collectiveSettings, hostSettings) => {
-  return get(collectiveSettings, 'expenseTypes.hasGrant', !get(hostSettings, 'disableGrantsByDefault', false));
+export const accountSupportsGrants = (account, host) => {
+  const accountHasGrantFlag = get(account, 'settings.expenseTypes.hasGrant');
+  if (isBoolean(accountHasGrantFlag)) {
+    // If the account's feature flag is set, use it as the main source of truth
+    return accountHasGrantFlag;
+  } else if (get(host, 'settings.disableGrantsByDefault')) {
```

Comment:
Maybe we should use `isBoolean` for this as well. Because if in the future someone decides to set `disableGrantsByDefault` to `true` and then later decides they want to enable grants by default so just goes to the db and sets it to `false`, then it won't work. They'll have to look at the code to realize that they need to remove the setting altogether. 🤔

Reply:
If they set `get(host, 'settings.disableGrantsByDefault')` to `false` then `get` will return `false` and we won't enter the `if` but rather fallback on the `else` below. I don't understand?

Reply:
`if (undefined)` has the same behavior than `if (false)`, both resolve to falsy

Reply:
> If they set `get(host, 'settings.disableGrantsByDefault')` to `false` then `get` will return `false` and we won't enter the `if` but rather fallback on the `else` below. I don't understand?

> `if (undefined)` has the same behavior than `if (false)`, both resolve to falsy

Ah I was a bit confused; you are right we are good here. 👍🏽